src/codem/preprocessing/preprocess.py

```diff
@@ -134,6 +136,21 @@ def resolution(self, value: float) -> None:
         self._resolution = value
         return None
 
+    def _PCS_from_GCS(
```

Comment:
This isn't a great name for this function. What you're doing here is computing a possible UTM zone in which to do your operations.